include/odp/api/spec/ipsec.h

```diff
@@ -1231,6 +1232,7 @@ typedef struct odp_ipsec_status_t {
  * Each successfully transformed packet has a valid value for these metadata
  * regardless of the inner packet parse configuration
  * (odp_ipsec_inbound_config_t):
+ * - IPv4 or IPv6 flag: Set packet flag according to packet contents.
```

Comment:
@psavol @JannePeltonen @muvarov @NikhilA-Linaro 
In most cases application will already have knowledge about payload protocol. So it is easier to provide such knowledge to IPsec machinery. So this is basically a question to @NikhilA-Linaro, If I got it right from SEC manual, your hardware also requires providing IP version, does it not?